com.unity.multiplayer.mlapi/Prototyping/NetworkNavMeshAgent.cs

```diff
@@ -10,6 +10,7 @@ namespace MLAPI.Prototyping
     /// A prototype component for syncing NavMeshAgents
     /// </summary>
     [AddComponentMenu("MLAPI/NetworkNavMeshAgent")]
+    [RequireComponent(typeof(NavMeshAgent))]
```

Comment:
Wouldn't this need to be ?
```suggestion
    [RequireComponent(typeof(NetworkNavMeshAgent))]
```

Reply:
No this should be `NavMeshAgent`. The `NetworkNavMeshAgent` component requires that there also is a `NavMeshAgent` on the same object. That's why we are using this attribute.

Reply:
Ah, this is referring to the `private NavMeshAgent m_Agent;` in `NetworkNavMeshAgent.cs`